REFERENCE.md

```diff
@@ -3524,7 +3524,7 @@ Struct[{
   ]],
   'TransmitHashPolicy'           => Optional[Enum['layer2', 'layer3+4', 'layer2+3', 'encap2+3', 'encap3+4']],
   'LACPTransmitRate'             => Optional[Enum['slow','fast']],
-  'MIIMonitorSec'                => Optional[Integer],
+  'MIIMonitorSec'                => Optional[Numeric],
```

Comment:
can you please add a unit test here that explicitly tests for a numeric, so people don't update it later to an integer again?